.github/workflows/dotnetBuild.yml

```diff
@@ -0,0 +1,36 @@
+name: CodingGuidelines Build
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+env:
+  version: 0.1.9
+jobs:
+  build:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET 5.0
+      uses: actions/setup-dotnet@v1
+      with:
+          dotnet-version: '5.0.x'
+    - name: Get Directory
+      run: pwd
+    - name: Get Dotnet Version
+      run: dotnet --version
+    - name: Dotnet info
+      run: dotnet --info
```

Comment:
These look like debugging steps. Are they needed?

Reply:
Decided it didn't hurt, we can remove them.

Reply:
Ill see if I can do it in one command. It does make it easier when a build does fail to check what is happening quickly, so I may keep it but simplify